lib/ovirt_metrics.rb

```diff
@@ -59,7 +59,8 @@ def self.host_realtime(host_id, start_time = nil, end_time = nil)
     host_realtime_metrics_to_hashes(metrics, nic_metrics)
   end
 
-  private
+  # The methods below this line are PRIVATE not meant to be used out of the scope
```

Comment:
exactly. But I removed the private because it made Rubocop rightful shout at me for adding a class method under "private". That's why I removed the "private" and added the comment.